app/assets/src/components/common/DetailsSidebar/BulkDownloadDetailsMode/AdvancedDownloadTab.jsx

```diff
@@ -43,13 +44,12 @@ export default class AdvancedDownloadTab extends React.Component {
       return "Failed to generate command. Please contact us for help.";
     }
 
-    return `curl -L "${bulkDownload.presigned_output_url}" > "${
-      bulkDownload.download_name
-    }.tar.gz"\
-        && mkdir -p "${bulkDownload.download_name}"\
-        && tar -zvxf "${bulkDownload.download_name}.tar.gz" -C "${
-      bulkDownload.download_name
-    }"\
+    let currentTimestamp = moment().format("MM-D-YYYY hh-mm-ssa");
+    let bulkDownloadFileName = `${bulkDownload.download_name} - ${currentTimestamp}`;
```

Comment:
nit: We should actually avoid spaces... but i guess the download name itself can come with spaces?

Reply:
Sounds good! I'll make the change && yes the download name comes with spaces already